resources/scss/views/ItemList/Components/Filter/item-category-dropdown.scss

```diff
@@ -0,0 +1,58 @@
+.list-controls {
+    #filterCollapse,.main-filters {
+        .page-content .card .fl-dropdown-container.fl-category-dropdown-container {
+            .fl-dropdown-label {
+                padding-left: 1rem;
+                text-overflow: ellipsis;
+                white-space: nowrap;
+                overflow: hidden;
+                display: inline-block;
+                width: 100%;
+                vertical-align: middle;
+            }
+
+            .fl-dropdown-content {
+                .fl-dropdown-item {
+                    &.form-check-label {
+                        display: flex;
+                        justify-content: space-between;
+                        flex-direction: row;
+                        align-items: center;
+
+                        .filter-badge {
+                            margin-right: 10px;
+                            right: unset;
+                            left: unset;
+                            top: unset;
+                            float: none;
+                            transform: unset;
```

Comment:
Why is this all required? Seems pretty odd.

Reply:
A comment would also be sufficient.

Reply:
To override the rules coming from someplace else.